src/AWS.Distro.OpenTelemetry.AutoInstrumentation/Plugin.cs

```diff
@@ -224,6 +224,26 @@ public void ConfigureTracesOptions(HttpClientTraceInstrumentationOptions options
                 this.ShouldSampleParent(activity);
             }
         };
+
+#if NETFRAMEWORK
+        options.FilterHttpWebRequest = request =>
+        {
+            if (request.RequestUri?.AbsolutePath == "/GetSamplingRules" || request.RequestUri?.AbsolutePath == "/SamplingTargets")
+            {
+                return false;
+            }
+
+            return true;
+        };
+
+        options.EnrichWithHttpWebResponse = (activity, request) =>
```

Comment:
Why not `EnrichWithHttpWebRequest`? If using the request method, this means that it will do the sampling when it's ready to send out a response and in this case, the parent sampling decision might be updated after the child has already been exported. 

I would also suggest wrapping the code above it with `#if !NETFRAMEWORK` since it only works on net core. 

Pasting this for reference: https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/main/src/OpenTelemetry.Instrumentation.Http/README.md#net-framework

Reply:
Good catch. It is a typo when clean up diff.